pkg/kudoctl/cmd/convert/convert.go

```diff
@@ -0,0 +1,173 @@
+package convert
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+
+	kudo "github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/vars"
+	"github.com/kudobuilder/kudo/pkg/util/helm"
+)
+
+// Run runs the convert command
+func Run(cmd *cobra.Command, args []string) error {
+	bundle, e := helm.ToBundle(vars.FrameworkImportPath)
+	if e != nil {
+		return e
+	}
+	if b := exists(vars.BundleOutputPath); b {
+		return fmt.Errorf("bundle output dir %v shouldn't exist", vars.BundleOutputPath)
+	}
+	e = os.Mkdir(vars.BundleOutputPath, 0755)
+	if e != nil {
+		return e
+	}
+
+	bundleParams := bundle.Parameters
+	bundle.Parameters = make([]kudo.Parameter, 0)
+	b, e := yaml.Marshal(bundle)
+	if e != nil {
+		return e
+	}
+	e = ioutil.WriteFile(fmt.Sprintf("%v/operator.yaml", vars.BundleOutputPath), b, os.ModePerm)
+	if e != nil {
+		return e
+	}
+	type P struct {
```

Comment:
could we go with more descriptive name? While reading this for the first time, I don't know what's P :/